packages/amplication-server/src/core/entity/constants.ts

```diff
@@ -100,7 +100,7 @@ type EntityData = Omit<
 
 export const DEFAULT_SINGLE_LINE_TEXT_MAX_LENGTH = 256;
 
-export const DEFAULT_ENTITIES: EntityData[] = [
+export const DEFAULT_USER_ENTITY: EntityData[] = [
```

Comment:
Why is this still an array then?

Reply:
because it required a big change e2e including the APIs and the client and I didn't want to include it